stardist/0.8.5/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM tensorflow/tensorflow:2.11.0-gpu-jupyter
+
+LABEL base_image="tensorflow:2.11.0-gpu-jupyter"
+LABEL version="1"
+LABEL maintainer="ko.sugawara@ens-lyon.fr"
+LABEL software="stardist"
+LABEL software.version="0.8.5"
+LABEL about.summary="Object Detection with Star-convex Shapes."
+LABEL about.home="https://github.com/stardist/stardist"
+LABEL about.license="BSD-3-Clause"
+LABEL about.license_file="https://github.com/stardist/stardist/blob/master/LICENSE.txt"
+
+MAINTAINER Florian Wuennemann <flowuenne@gmail.com>
+
+ARG DEBIAN_FRONTEND="noninteractive"
+ARG STARDIST_VERSION="0.8.5"
+ARG NVIDIA_DRIVER_VERSION=470
```

Comment:
I feel like this will be an issue? According to the stardist github, the NVIDIA_DRIVER_VERSION should be adjusted to the runtime environnement (depending on the GPU you have, I guess).

No sure how the container will behave if the runtime env does not match

Reply:
Thanks for the review @mboudet ! 

What would you suggest how to handle this issue?

Reply:
No quite sure, to be honest. It would be possible to install the correct lib at runtime I guess, but it would be quite dirty, and would probably not work with singularity.

I'm not sure how does it work with the conda package? Would it be possible to directly install the package in the dockerfile (like [here](https://github.com/BioContainers/containers/blob/07af7edbedc8fd6e5b60895ca6d9bd21b1aa71dd/megahit/1.2.9/Dockerfile#L26)), or does it require external librairies ?

Reply:
I don't think I understand what you mean whether it would be possible to directly install the package in the dockerfile? Which package are you referring to? I don't see anything related to nvidia drivers in the recipe you linked unforunately? 🤷🏼

Reply:
Well, you use the 'NVIDIA_DRIVER_VERSION' variable to install a specific version of the libnvidia-compute in the next step. (Line 24). According the the stardist github, the libnvidia-compute version should match the GPU driver version.

Reply:
So as is custom these days, I asked ChatGPT for help. The suggestion from it was to specify `--build-arg` when building the docker container at runtime to adjust the ENV variable `NVIDIA_DRIVER_VERSION` to the one available:
`docker build --build-arg NVIDIA_DRIVER_VERSION=<desired_version> -t <your_image_name> .
`

I am not sure there is a good way to automatically detect this version number. I also don't really know whether we could build a container for cpu completely without NVIDIA drivers...

Reply:
Yeah, this would work, but that mean that the user would need to build the image themselve (so, not much point in having the image on biocontainers). Also, building an image usually require root/sudo privileges, which are not usually available in computing clusters.

My question regarding the conda package stands though. I'm not sure if a mamba/conda install in the container would work. This might need input from the software's developpers.

Reply:
I asked them in https://github.com/stardist/stardist/issues/259